### ISSUE TITLE
[ feature request ] - Full Featured Event Organizing tool

### ISSUE THREAD

**Issue by naturalgeek** [friendica/friendica#2083]
Prologue:

Currently, there is only one way to create events on friendica using ical.
But for my opinion, this is more a calendar application which creates calendar entries than a event organisation tool.
The only reason why my friends still using fakebook is on one hand the seamless chat functionality and - and this is a bigger thing - the event organisation tool.
We are using this feature very often but on friendica its almost unusable for management!

Feature:

Creating event should be done in a group. If someone organize an event, he want t invite people, he want to see who is joining and the guest want to share informations, pictures and videos.
Since we organize Parties for electronic music often, we would really like to use friendica and my friends would like to join but its not possible without a feature like this.
There are masses of people, which uses Social Networks not to post their entire life but  to communicate in private or in groups and organize event and invite their friends.
This i currently not possible in a good way which is very sad.

**Comment by annando** [comment]
In the develop branch we have the possibility that user can say if they attend an event or not. This doesn't cover everything of your requests but is a start, I think.

**Comment by tobiasd** [comment]
> Currently, there is only one way to create events on friendica using ical.

I don't think we can create events via ical--can we? There is an addon which let you export your events as ical or csv file.

For creating a new event, you simply go to your events overview page and create a new one (see also https://github.com/friendica/friendica/blob/master/doc/events.md). The event is then distriuted to all your contacts, or only a selected audience, just like a regular posting. They will see it in their network stream and on their events page.

In current master branch you can only (dis-)like and comment on events, but in develop branch (which should be released mid-December) you can also attend/don't attend/maybe attend an event.
